scripts/in_container/check_environment.sh

```diff
@@ -100,7 +100,7 @@ function resetdb_if_requested() {
 function startairflow_if_requested() {
     if [[ ${START_AIRFLOW:="false"} == "true" ]]; then
 
-	. "$( dirname "${BASH_SOURCE[0]}" )/configure_environment.sh"
+    . "$( dirname "${BASH_SOURCE[0]}" )/configure_environment.sh"
```

Comment:
Please rebase on to latest master -- this has already been fixed there.

Reply:
Done